packages/xstate-store/src/types.ts

```diff
@@ -382,14 +384,18 @@ export interface AtomOptions<T> {
   compare?: (prev: T, next: T) => boolean;
 }
 
+export interface AsyncAtomOptions<T> extends AtomOptions<T> {
+  signal?: AbortSignal;
+}
+
 export type AnyAtom = BaseAtom<any>;
 
 export interface InternalReadonlyAtom<T>
   extends InternalBaseAtom<T>,
     Dependency,
     Subscriber {
   /** @internal */
-  _update(): boolean;
+  _update(getValue?: () => T): boolean;
```

Comment:
This is already available on `InternalBaseAtom` and this interface extends from that. So you could remove it from here

Reply:
Yes, `getValue` is a way of "setting" a value of a derived atom, while tracking new dependencies. I'll unify